docs/development.md

```diff
@@ -0,0 +1,38 @@
+# Setting Up a Development Environment
+
+
+## Table of contents
+
+1. [Automatically format code and check for code styling](#python-styling)
+
+## Automatically format code and check for code styling <a name='python-styling'></a>
+
+Presidio services are PEP8 compliant and continuously enforced on style guide issues during the build process using `flake8`.
+
+Running flake8 locally, using `pipenv run flake8`, you can check for those issues prior to committing a change.
+
+To make things easier, you can use pre-commit hooks to verify and automatically format code upon a git commit, using `black`:
+
+1. [Install pre-commit package manager locally.](https://pre-commit.com/#install) 
```

Comment:
If it's just `pip install pre-commit` then I would consider writing it here.

Reply:
You can choose, its pip install in your own venv, its brew, its curl to the installation python file.

Reply:
how do we know someone actually ran this before the commit? 
Is there going to be some check in the pipeline?

Reply:
We can't, and we can't force to use it, it's suppose to help us find and fix issues before the build (which will still fail regardless of using pre-commit)